### PR TITLE
Update BTCPayVsOthers.md

### PR DESCRIPTION
**Pull request by syndicato-de-gato**
fixed some grammar and typos

### CODE REVIEW

File: BTCPayVsOthers.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Most new merchants will likely only consider the price of the service. Since BTCPay is free, that may have led you here and if so, welcome.
 
-BTCPay is a code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+BTCPay is code, not a company. There is no third-party between a merchant and a customer. The merchant is always in full control of their funds. There's no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
```
Comment (Kukks) [inline BTCPayVsOthers.md]: `There are no processing or subscription fees` made more sense to me?

File: BTCPayVsOthers.md [hunk 2]
```diff
@@ -31,14 +31,15 @@ Every payment processor has features, here are some BTCPay features:
 * **Payment Buttons** - HTML embeddable donation and pay buttons.
 * **Unlimited Stores** - Merchants can process payments for their own stores, or for others.
 * **Translations** - Customers can pay in 20 different languages.
+* **Payment Requests** - Create & send an invoice requesting payment for goods or services.
```
Comment (Kukks) [inline BTCPayVsOthers.md]: `Create & send a long-lived invoice requesting payment for goods or services. `?

File: BTCPayVsOthers.md [hunk 3]
```diff
@@ -90,21 +91,21 @@ Many payment processors claim to have no middleman. Saying the funds go directly
 * If the payment processor has terms and conditions stating they can hold or freeze your account.
 * Fees for using the payment processor are automatically taken out from the customer's payment to the merchant.
 
-Payment processors perform some middleman actions by using a **custodial wallet**. A payment processor can use an internal custodial wallet for altering customer payments before routing them to merchants. This is how they can collect fees, hold payments for verification and processing, etc. This type of wallet is an intermediary between the merchant wallet and the customer wallet. It's the middleman wallet.
+Payment processors are able act as a middleman by using a **custodial wallet**. A payment processor can use an internal custodial wallet for altering customer payments before routing them to merchants. This is how they can collect fees, hold payments for verification and processing, etc. This type of wallet is an intermediary between the merchant wallet and the customer wallet. It's the middleman wallet.
```
Comment (Kukks) [inline BTCPayVsOthers.md]: Payment processors are able to act as a middleman

File: BTCPayVsOthers.md (hunk 1 above)
Comment (syndicato-de-gato) [reply BTCPayVsOthers.md]: Go with the version you prefer.

File: BTCPayVsOthers.md (hunk 3 above)
Comment (syndicato-de-gato) [reply BTCPayVsOthers.md]: could change to "Payment processors are able to act as middlemen"

**Pull request merged**